index.html

```diff
@@ -26,16 +26,131 @@ <h1 class="primaryHeading">
         <p class="mainContent">
             I'm a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don't hestiate to contact me.
         </p>
-
         <a href="#" class="connect">Let's Connect</a>
         <ul class="icons">
-           <li><a href="#" class="iconList"><i class="fa-brands fa-twitter"></i></a></li>
-           <li><a href="#" class="iconList"><i class="fa-brands fa-linkedin-in"></i></a></li>
-           <li><a href="#" class="iconList"><i class="fa-brands fa-github"></i></a></li>
-           <li><a href="#" class="iconList"><i class="fa-brands fa-instagram"></i></a></li>
+            <li><a href="#" class="iconList"><i class="fa-brands fa-twitter"></i></a></li>
+            <li><a href="#" class="iconList"><i class="fa-brands fa-linkedin-in"></i></a></li>
+            <li><a href="#" class="iconList"><i class="fa-brands fa-github"></i></a></li>
+            <li><a href="#" class="iconList"><i class="fa-brands fa-instagram"></i></a></li>
         </ul>
 
     </main>
-    
+
+    <section class="workSection">  
+        <div class="card">
+            <div class="image2 img"></div>
+            <h1 class="tonic">Tonic</h1>
+
+            <div class="divList">
+                <p class="canopy">canopy</p>
```

Comment:
- [ ] Kindly use hx for this as it has semantic meaning for devices such as screen readers. It is a subtitle not a paragraph